x-pack/plugins/security_solution/server/endpoint/routes/metadata/handlers.ts

```diff
@@ -191,6 +185,9 @@ export function getMetadataListRequestHandlerV2(
       didUnitedIndexError = true;
     }
 
+    const { endpointResultListDefaultPageSize, endpointResultListDefaultFirstPageIndex } =
```

Comment:
BTW - I have no idea why we ever added this to the Security Solution plugin configuration (@pzl do you?)

if you all can't identify a reason, can we open an issue to maybe remove it in 8.0 (since we can break backwards compatibility)?

Reply:
no reason or actual usage comes to mind. :axe:  it

Reply:
https://github.com/elastic/security-team/issues/2300

Reply:
created ticket.